src/ray/core_worker/test/scheduling_queue_test.cc

```diff
@@ -63,11 +71,16 @@ TEST(SchedulingQueueTest, TestTaskEvents) {
   instrumented_io_context io_service;
   MockWaiter waiter;
   MockTaskEventBuffer task_event_buffer;
+
+  std::vector<ConcurrencyGroup> concurrency_groups{ConcurrencyGroup{"io", 1, {}}};
+  auto pool_manager =
+      std::make_shared<ConcurrencyGroupManager<BoundedExecutor>>(concurrency_groups);
+
   ActorSchedulingQueue queue(io_service,
                              waiter,
                              task_event_buffer,
-                             std::make_shared<ConcurrencyGroupManager<BoundedExecutor>>(),
-                             std::make_shared<ConcurrencyGroupManager<FiberState>>(),
+                             pool_manager,
+                             /*fiber_state_manager=*/nullptr,
```

Comment:
After #51069 is merged, it will be impossible for both `pool_manager` and `fiber_state_manager` to be non-null simultaneously. Hence, set `fiber_state_manager` to nullptr.